tests/rewards/test_reward_nets.py

```diff
@@ -113,11 +169,29 @@ def test_init_no_crash(
         )
 
 
+@pytest.mark.parametrize("env_name", IMAGE_ENVS)
+@pytest.mark.parametrize("reward_net_cls", MAKE_IMAGE_REWARD_NET)
+@pytest.mark.parametrize("reward_net_kwargs", REWARD_NET_KWARGS)
```

Comment:
I think normalizing image-based observations is a bit of a weird thing to do, but it seems reasonable to test it given we're trying to support it. (Might also be fine to just explicitly throw an error if people try to do this, and test for that.)